<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormModificationsGroups</class>
 <widget class="QWidget" name="FormModificationsGroups">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Группы модификаций</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitterModificationsGroups">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>3</number>
     </property>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButtonAddGroup">
           <property name="toolTip">
            <string>Добавить группу для параметров</string>
           </property>
           <property name="text">
            <string>Добавить</string>
           </property>
           <property name="icon">
            <iconset resource="../../res.qrc">
             <normaloff>:/32/icons/32x32/add.png</normaloff>:/32/icons/32x32/add.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonEditGroup">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Редактировать группу параметров</string>
           </property>
           <property name="icon">
            <iconset resource="../../res.qrc">
             <normaloff>:/32/icons/32x32/edit.png</normaloff>:/32/icons/32x32/edit.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonDelGroup">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Удалить выбранные группы</string>
           </property>
           <property name="icon">
            <iconset resource="../../res.qrc">
             <normaloff>:/32/icons/32x32/delete.png</normaloff>:/32/icons/32x32/delete.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonRefresh">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Обновить список групп параметров</string>
           </property>
           <property name="icon">
            <iconset resource="../../res.qrc">
             <normaloff>:/32/icons/32x32/refresh.png</normaloff>:/32/icons/32x32/refresh.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonGroupUp">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Переместить группу выше</string>
           </property>
           <property name="icon">
            <iconset resource="../../res.qrc">
             <normaloff>:/32/icons/32x32/up.png</normaloff>:/32/icons/32x32/up.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonGroupDown">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Переместить группу ниже </string>
           </property>
           <property name="icon">
            <iconset resource="../../res.qrc">
             <normaloff>:/32/icons/32x32/down.png</normaloff>:/32/icons/32x32/down.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonLoadFromCsv">
           <property name="toolTip">
            <string>Загрузить типы товаров из CSV файлов</string>
           </property>
           <property name="text">
            <string>Загрузить из CSV</string>
           </property>
           <property name="icon">
            <iconset resource="../../res.qrc">
             <normaloff>:/32/icons/32x32/csv-32.png</normaloff>:/32/icons/32x32/csv-32.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>43</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonSelect">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Выбрать группу для модификаций</string>
           </property>
           <property name="text">
            <string>Выбрать</string>
           </property>
           <property name="icon">
            <iconset resource="../../res.qrc">
             <normaloff>:/64/icons/64x64/select.png</normaloff>:/64/icons/64x64/select.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelHelp">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выделите группу модификаций или несколько смежных групп (удерживая клавишу &amp;lt;Shift&amp;gt;) и нажмите кнопку: &lt;img src=&quot;:/64/icons/64x64/select.png&quot; height=&quot;16&quot;/&gt;Выбрать&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeViewGroupsM"/>
       </item>
      </layout>
     </widget>
     <widget class="FormModificationsGroupsValues" name="widgetValues" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionAddGroup">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/32/icons/32x32/add.png</normaloff>:/32/icons/32x32/add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="toolTip">
    <string>Создать новый набор (группу) для модификаций</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actionEditGroup">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/32/icons/32x32/edit.png</normaloff>:/32/icons/32x32/edit.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
   <property name="toolTip">
    <string>Редактировать выбранную группу</string>
   </property>
  </action>
  <action name="actionRemoveGroup">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/32/icons/32x32/delete.png</normaloff>:/32/icons/32x32/delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="toolTip">
    <string>Удалить выбранные группы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/32/icons/32x32/refresh.png</normaloff>:/32/icons/32x32/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="toolTip">
    <string>Обновить данные</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormModificationsGroupsValues</class>
   <extends>QWidget</extends>
   <header>formmodificationsgroupsvalues.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
